docs/releases/h_2_6.md

```diff
@@ -0,0 +1,26 @@
+# iRobot® Create® 3 Release H.2.6
+[[Click here to download release H.2.6]](https://edu.irobot.com/create3/firmware/H.2.6)
+
+!!! warning
+    When using Fast-DDS, startup times are about 30s longer than in our Galactic release.
+
+## This release is running ROS 2 Humble with the following interface library versions:
+
+- [irobot_create_msgs - 2.1.0](https://github.com/iRobotEducation/irobot_create_msgs/tree/2.1.0)
+- [cyclonedds - 0.10.3](https://github.com/eclipse-cyclonedds/cyclonedds/tree/0.10.3)
+- [Fast-DDS - 2.6.4](https://github.com/eProsima/Fast-DDS/tree/2.6.4)
+
+## Release Overview
+For ROS 2[^1] users, this is a bugfix release.
+For iRobot® Education Bluetooth[^2] users, there are no changes.
+See below for details.
+
+## Changelog (from H.2.5)
```

Comment:
Since we are releasing back-to-back, most users will likely jump on this and skip 2.5.
Should we remind here that 2.5 made a lot of changes and link to that changelog?